lib/streamlit/elements/alert.py

```diff
@@ -12,86 +12,147 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-from typing import cast, TYPE_CHECKING
+from typing import cast, Optional, TYPE_CHECKING
 
 from streamlit.proto.Alert_pb2 import Alert as AlertProto
+from streamlit.errors import StreamlitAPIException
+from streamlit.string_util import is_emoji_valid
```

Comment:
nits: would be good to have these sorted alphabetically (errors / proto / string_util)

Reply:
Updated in [c675117](https://github.com/streamlit/streamlit/pull/4964/commits/c67511753c61c0d0e15f8454c78e3c0be9cdc3de)